src/test/functional/functional-tests.js

```diff
@@ -14,75 +14,74 @@
  * limitations under the License.
  */
 
-import { Client, Policy } from '../main/minio.js'
-import * as Minio from '../main/minio.js'
-import os from 'os'
-import stream from 'stream'
-import crypto from 'crypto'
-import async from 'async'
-import _ from 'lodash'
-import fs from 'fs'
-import http from 'http'
-import https from 'https'
-import url from 'url'
-import { assert } from 'chai'
-import superagent from 'superagent'
+const minio = require('minio');
+const Policy = minio.Policy;
+const Client = minio.Client;
+const os = require('os');
+const stream = require('stream');
+const crypto = require('crypto');
+const async = require('async');
+const _ = require('lodash');
+const fs = require('fs');
+const http = require('http');
+const https = require('https');
+const url = require('url');
+const chai = require('chai');
```

Comment:
```
import { assert } from 'chai'
```

this is an often used pattern when you need to use one or two functions, how come it was removed?